profile/README.md

```diff
@@ -0,0 +1,48 @@
+<div align="center" style="background-color: white;">
+<a href="https://www.transportforthenorth.com/">
+<img src="https://www.transportforthenorth.com/wp-content/themes/tfn-theme/img/logo.svg"
+  alt="Transport for the North logo">
+</a>
+</div>
+
+# About Us
+
+Transport for the North (TfN) are England's first Sub-national Transport Body (STB). We were formed to transform the transport system across the North of England, providing the infrastructure needed to drive economic growth.
+TfN's role is to add strategic value by ensuring that funding and strategy decisions about transport in the North are informed by local knowledge and requirements, more details on our work can be found on our website at [transportforthenorth.com](https://www.transportforthenorth.com/).
+
+# Open Source Analytics
+
+TfN has a focus on openness and transparency and as such many of our analytics tools and processes are shared openly on GitHub for scrutiny and use by others.
+The repositories fall into one of two categories, either internal TfN or common analytical framework.
+
+## TfN
+
+These are processes and tools which are primarily used within TfN for our various data analytics tasks, they're shared to allow for others to benefit from our work and view our analytics.
+The tools and process that fall into this category are generally more specific to the transport analytics needs of the North of England and providing an evidence base to aid in decisions in and around North England.
+
+| Repository | Description |
+| :--------: | :---------- |
+| [NorMITs-Demand](https://github.com/Transport-for-the-North/NorMITs-Demand) | TfN's mainland GB transport demand model. |
+
+## Common Analytical Framework (CAF)
+
+The Common Analytical Framework (often shortened to CAF) is a collaboration between transport bodies in the UK to develop and maintain commonly used transport analytics and appraisal tools.
+The tools and processes that fall into this category generally are branded as "caf.X" and are built in collaboration with other transport bodies and are aimed to be more generic and flexible processes which others can pickup and use in their analytics.
+[CAF Handbook](https://transport-for-the-north.github.io/CAF-Handbook) is a guide which provides more details on CAF and the various tools and process.
+
+| Repository | Description |
+| :--------: | :---------: |
+| [CAF.toolkit](https://github.com/Transport-for-the-North/caf.toolkit) | CAF.Toolkit focusses on generic tools and functions that are used across the CAF. |
```

Comment:
Swapping these sections round - I think its easier to explain the non-CAF tools once we've described CAF

```suggestion
## Common Analytical Framework (CAF)

The Common Analytical Framework (CAF) is a collaboration between transport bodies in the UK to develop and maintain commonly used transport analytics and appraisal tools.
The tools and processes that fall into this category generally are branded as "caf.X" and are built in collaboration with other transport bodies. CAF tools are generic and flexible processes which allow others to pick up and use in their analytics. The CAF has been built in many smaller modules to allow a range of use-cases, from taking an entire model as is, to selecting just to relevant components.
For further information on the CAF, please see the [CAF Handbook](https://transport-for-the-north.github.io/CAF-Handbook), this is a guide that details the CAF and its various tools and process.

| Repository | Description |
| :--------: | :---------: |
| [CAF.toolkit](https://github.com/Transport-for-the-North/caf.toolkit) | CAF.Toolkit focusses on generic tools and functions that are used across transport analysis. |

## TfN

Internal TfN tools are those which we use internally, but we have not migrated into the CAF. These tend to be tools that we have had some external interest in, but lack the resource to generalise at this time.
We use and develop these tools within TfN, sharing them to allow for others to benefit from our work, view our analytics, and maybe build upon them in their own work.
The tools and process that fall into this category are sometimes more specific to the transport analytics needs of the North of England.

| Repository | Description |
| :--------: | :---------- |
| [NorMITs-Demand](https://github.com/Transport-for-the-North/NorMITs-Demand) | TfN's mainland GB transport demand model. |

```